### PR TITLE
Fix Nightly build tests (x-browser & full functional)

### PR DESCRIPTION
**Pull request by gokul-sol**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/SSCSSI-224

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: test/e2e/e2e-sya/e2e.attendanceAllowance.en.test.js [hunk 1]
```diff
@@ -11,6 +11,6 @@ const hasDwpIssuingOffice = testDataEn.benefitTypes[5].hasDwpIssuingOffice;
 
 Feature(`${language.toUpperCase()} - Attendance Allowance E2E SYA - Full Journey`);
 
-Scenario(`${language.toUpperCase()} - ${benefitCode} E2E SYA Journey @fullFunctional @e2e`, ({ I }) => {
+Scenario(`${language.toUpperCase()} - ${benefitCode} E2E SYA Journey @fullFunctional @e2e @fullFunctional-today`, ({ I }) => {
```
Comment (pats-john) [inline test/e2e/e2e-sya/e2e.attendanceAllowance.en.test.js:14]: Possibly -today is not the right tag name.

**Review by pats-john — APPROVED** [review]

File: test/e2e/e2e-sya/e2e.attendanceAllowance.en.test.js (hunk 1 above)
Comment (gokul-sol) [reply test/e2e/e2e-sya/e2e.attendanceAllowance.en.test.js]: good catch @pats-john , removed it.

File: config/default.json [hunk 2]
```diff
@@ -31,7 +31,8 @@
     "waitForAction": "500",
     "outputDir": "./functional-output",
     "smokeOutputDir": "./smoke-output",
-    "frontendUrl": "https://localhost:3000"
+    "frontendUrl": "http://localhost:3000",
+    "retry": 8
```
Comment (nitinprabhuhmcts) [inline config/default.json:35]: do you need 8 retries ?

File: test/e2e/page-objects/compliance/dwpIssuingOffice.js [hunk 3]
```diff
@@ -1,17 +1,18 @@
 function enterDWPIssuingOfficeAndContinue(commonContent, id) {
   const I = this;
 
-  I.wait(1);
-  I.scrollPageToBottom();
-  I.wait(1);
+  // I.wait(1);
+  // I.scrollPageToBottom();
```
Comment (nitinprabhuhmcts) [inline test/e2e/page-objects/compliance/dwpIssuingOffice.js:5]: can we plz remove commented code ?

File: test/e2e/page-objects/compliance/dwpIssuingOffice.js [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+  I.wait(3);
   I.selectOption({ id: 'pipNumber' }, id);
   I.click(commonContent.continue);
 }
 
 function enterDWPIssuingOfficeAndContinueAfterSignIn(commonContent, id) {
   const I = this;
 
-  I.scrollPageToBottom();
+  // I.scrollPageToBottom();
```
Comment (nitinprabhuhmcts) [inline test/e2e/page-objects/compliance/dwpIssuingOffice.js:14]: same here

File: test/e2e/page-objects/compliance/dwpIssuingOffice.js [hunk 5]
```diff
@@ -20,7 +21,8 @@ function enterDWPIssuingOfficeAndContinueAfterSignIn(commonContent, id) {
 function enterDWPIssuingOffice(commonContent, id) {
   const I = this;
 
-  I.scrollPageToBottom();
+  // I.scrollPageToBottom();
```
Comment (nitinprabhuhmcts) [inline test/e2e/page-objects/compliance/dwpIssuingOffice.js:24]: same here

File: test/e2e/page-objects/identity/appellantDetails.js [hunk 6]
```diff
@@ -2,61 +2,80 @@ const appellant = require('test/e2e/data.en').appellant;
...
 }
 
-function enterAppellantDOBAndContinue(commonContent, day, month, year) {
+function enterAppellantDOBAndContinue(language, commonContent, day, month, year) {
   const I = this;
+  const appellantDOBContent = language === 'en' ? appellantDOBContentEn : appellantDOBContentCy;
 
-  I.wait(1);
+  I.waitForText(appellantDOBContent.title.withoutAppointee);
+  // I.waitForElement('input[name*="day"]', 3);
```
Comment (nitinprabhuhmcts) [inline test/e2e/page-objects/identity/appellantDetails.js:44]: we can remove commented code unless it is temporary ?

File: test/e2e/page-objects/identity/appellantDetails.js [hunk 7]
```diff
@@ -2,61 +2,80 @@ const appellant = require('test/e2e/data.en').appellant;
...
 }
 
-function enterAppellantNINOAndContinue(commonContent, nino) {
+function enterAppellantNINOAndContinue(language, commonContent, nino) {
   const I = this;
+  const appellantNINOContent = language === 'en' ? appellantNINOContentEn : appellantNINOContentCy;
 
-  I.wait(1);
+  I.waitForText(appellantNINOContent.title.withoutAppointee);
+  // I.waitForElement('#nino', 3);
```
Comment (nitinprabhuhmcts) [inline test/e2e/page-objects/identity/appellantDetails.js:68]: same here

File: test/e2e/page-objects/compliance/dwpIssuingOffice.js (hunk 3 above)
Comment (gokul-sol) [reply test/e2e/page-objects/compliance/dwpIssuingOffice.js]: @nitinprabhuhmcts  done

File: test/e2e/page-objects/compliance/dwpIssuingOffice.js (hunk 4 above)
Comment (gokul-sol) [reply test/e2e/page-objects/compliance/dwpIssuingOffice.js]: @nitinprabhuhmcts  done

File: test/e2e/page-objects/compliance/dwpIssuingOffice.js (hunk 5 above)
Comment (gokul-sol) [reply test/e2e/page-objects/compliance/dwpIssuingOffice.js]: @nitinprabhuhmcts  done

File: test/e2e/page-objects/identity/appellantDetails.js (hunk 6 above)
Comment (gokul-sol) [reply test/e2e/page-objects/identity/appellantDetails.js]: @nitinprabhuhmcts  done

File: test/e2e/page-objects/identity/appellantDetails.js (hunk 7 above)
Comment (gokul-sol) [reply test/e2e/page-objects/identity/appellantDetails.js]: @nitinprabhuhmcts  done

**Review by gokul-sol — COMMENTED** [review]
Review comments are addressed now, thanks.

**Review by nitinprabhuhmcts — APPROVED** [review]

**Pull request merged**